apps/teams-test-app/e2e-test-data/store.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Store",
+  "platforms": "Web",
+  "checkIsSupported": {
+    "domElementName": "checkCapabilityStore"
```

Comment:
You want to add a `"hostSdkVersion"` parameter to this as well, but I'm less sure about that. Please follow up with @Kangxuan-Ye

Reply:
updated